inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImpl.java

```diff
@@ -159,6 +159,23 @@ public void registerKnowledgeBase(KnowledgeBase kb, RepositoryImplConfig cfg)
         repoManager.addRepositoryConfig(new RepositoryConfig(repositoryId, cfg));
         entityManager.persist(kb);
     }
+    
+    @Override
+    public void defineBaseProperties(KnowledgeBase kb, boolean kbUpdateFlag) 
+    {
+        // KB will initialize base properties with base IRI schema properties defined by user
+        if (kb.getType() == RepositoryType.LOCAL) {
+            KBProperty property = new KBProperty(kb.getSubclassIri().getLocalName());
+            property.setIdentifier(kb.getSubclassIri().stringValue());
+            createBaseProperty(kb, property);
+            property = new KBProperty(kb.getLabelIri().getLocalName());
```

Comment:
Please avoid re-purposing variables. Anyway, when the constructor mention above has been introduced, this variable is not required anymore.